utils/vpr_fixup_post_synth.py

```diff
@@ -76,6 +76,117 @@ def sub_func(match):
 # =============================================================================
 
 
+def merge_verilog_ports(code):
```

Comment:
Something has gone seriously wrong for us to have this script in the first place. Why didn't we just fix VPR's output directly?

Reply:
The problem is that VPR does not understand the concept of multi-bit top-level design ports. Internally each individual one is represented as a `.input` / `.output` cell whit a single pin. In some cases we want them to be represented as multi-bit in post-layout Verilog netlist and SDF timing annotation files hence the script.

The scripts solves the problem reported by QuickLogic - there was a need to have matching ports between an input design and a post-layout netlist. The proper fix including modifying VPR output has not been implemented yet.

Reply:
Let's fix VPR then?

Reply:
I've created a feature request in VtR for tracking this issue: https://github.com/verilog-to-routing/vtr-verilog-to-routing/issues/1946

Reply:
Opened PR in vtr that implements machine readable summary outputs for timings and block usage: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1947